tests/integration/slashing/keeper/keeper_test.go

```diff
@@ -4,65 +4,133 @@ import (
 	"testing"
```

Comment:
maybe one test in here should be using the unjail message in the message server. currently i dont see one

Reply:
we already have tests for `Unjail` function from msg server in the `x/slashing/keeper/msg_server_test.go`.
why do we need tests here again ?